pkg/controller/redirect/index.go

```diff
@@ -18,20 +18,62 @@ import (
 	"fmt"
 	"net/http"
 	"strings"
+	"time"
 
 	"github.com/google/exposure-notifications-verification-server/pkg/controller"
 )
 
+// cacheTTL is the amount of time for which to cache these values.
+const cacheTTL = 30 * time.Minute
```

Comment:
best to make this part of config w/ default of 30m so runtime configurable

Reply:
done